src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/events/PluginPatchsetCreatedEvent.java

```diff
@@ -109,6 +112,10 @@ public boolean shouldTriggerOn(GerritTriggeredEvent event) {
         if (!super.shouldTriggerOn(event)) {
             return false;
         }
+        // if it was a manually triggered event, always honor it
```

Comment:
Well you might have one job that triggers on drafts (hidden from unknowing people) and one that triggers on non drafts.
If you then select a patch set that isn't a draft that would cause both jobs to trigger and that's probably not what you want.